### PR TITLE
NAS-109548 / 12.0 / Use progress bar for snapshot multi delete

### PR DESCRIPTION
**Pull request by dszidi**
Tested against INTERNAL 153. This PR offers the following improvements

- Multi delete now uses a progress bar via entity-job
- Third parameter {0} added as per advice from middleware team on ticket
- After snapshots have been deleted, the table properly reloads

### CODE REVIEW

File: src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts [hunk 1]
```diff
@@ -253,8 +265,11 @@ export class SnapshotListComponent {
         this.entityList.loader.open();
         this.entityList.loaderOpen = true;
         this.ws.call(this.wsDelete, [item.name]).subscribe(
-          (res) => { this.entityList.getData() },
+          (res) => { 
+            this.entityList.getData() 
+          },
           (res) => {
+            console.warn(res);
```
Comment (ryousaf) [inline src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts:272]: Is this left here intentionally?

[the remark this answers is not in the document]
Comment (dszidi) [reply src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts:272]: Debug. Will take it out

File: src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts [hunk 2]
```diff
@@ -263,6 +268,38 @@ export class SnapshotListComponent {
...
+    dialogRef.componentInstance.setCall(this.wsMultiDelete, params);
+    dialogRef.componentInstance.progressNumberType = 'nopercent';
+    dialogRef.componentInstance.submit();
+
+    dialogRef.componentInstance.success.subscribe((job_res) => {
+      dialogRef.close();
+      this.entityList.getData();
+      this.entityList.selected = [];
+
+      let infoMessage: string = T('Deleted ') + params[1].length;
```
Comment (william-gr) [inline src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts:292]: Having leading/trailing spaces as part of the translation string is wrong, no?

Comment (dszidi) [reply src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts]: Updated

**Review by william-gr — APPROVED** [review]

**Review by denysbutenko — APPROVED** [review]
Looks good

**Comment by william-gr** [comment]
Thanks @denysbutenko

**Pull request merged**